misc/ls-extensions/modules/json-to-record-converter/build.gradle

```diff
@@ -36,7 +36,11 @@ dependencies {
         exclude group: "org.slf4j", module: "slf4j-ext"
         exclude group: "javax.validation", module: "validation-api"
     }
-    compileOnly(group: 'org.eclipse.lsp4j', name: 'org.eclipse.lsp4j', version: '0.8.1')
+    compileOnly (group: 'org.eclipse.lsp4j', name: 'org.eclipse.lsp4j', version: '0.8.1') {
```

Comment:
This is compile Only apparently guava is coming from `swagger-parser-v2-converter` dependency. Shall we exclude from there?

Reply:
Changed in e96fe26